app/views/application/_messages.html.erb

```diff
@@ -1,6 +1,3 @@
 <% if flash[:notice] %>
   <p class="flash-notice"><%= flash[:notice] %></p>
 <% end %>
-<% if flash[:error] %>
```

Comment:
And here's where your rails knowledge gets the best of me :) I've been using :error, but it appears rails expects :alert. Cool.